modular_skyrat/modules/contractor/code/items/boxes.dm

```diff
@@ -73,4 +73,49 @@
 	new /obj/item/extraction_pack/contractor(src)
 	new /obj/item/fulton_core(src)
 
+/obj/item/storage/box/syndicate/contract_kit/midround
+	name = "Contract Kit"
+	special_desc = "Supplied to Syndicate contractors."
+	special_desc_requirement = EXAMINE_CHECK_CONTRACTOR
+	icon_state = "syndiebox"
+	illustration = "writing_syndie"
+
+/obj/item/storage/box/syndicate/contract_kit/PopulateContents()
+	// All about 4 TC or less - some nukeops only items, but fit nicely to the theme.
+	var/list/item_list = list(
```

Comment:
Make it a static list, since it doesn't really need to be instanced every time.

Reply:
the code below uses `pick_and_take`, instead of just `pick`